### DISCUSSION TITLE
Is there a way to change the color of text in the `InlineCode` component?

### DISCUSSION THREAD

**Discussion by Danii2020** [twilio-labs/paste#3283] [Q&A] [answered]
Hi team! I was wondering to change the text color of the `InlineCode` component, I want to change from the default black color to a red one, is it any way to do this in the component?

Thanks in advance!

**Comment by SiTaggart** [comment, marked as answer]
Hi there

We're pretty intentional about the restrictions we place on the visual appearance of our components, mainly to create visual consistency across Twilio products. We want Code, as an example, to be represented the same no matter what product or Twilio experience you are using, so we don't allow components to be customized directly.

For customers using Paste to build their own experiences, or products that are external to Core R&D teams, we do provide a [Customization Provider](https://paste.twilio.design/customization) which comes with a number of ways to change the look and feel of Paste globally and by component. 

This shouldn't be used on core Twilio products, but we understand that customers and special projects eternal to Core R&D might require this functionality.

Hope that helps

**Reply by Danii2020** [reply]
Hi Simon, thank you for the confirmation, my team and I will stick with the default `InlineCode` default styles.
